cost-analyzer/values-eks-cost-monitoring.yaml

```diff
@@ -1,192 +1,26 @@
-global:
-  grafana:
-    enabled: false
```

Comment:
We should call out changes in helm release notes, but this is a great change IMO. thanks @chipzoller

Reply:
I don't see any changes what our defaults were.

You good with this @dwbrown2 ? ideally get this in 2.2 as we are updating docs that will reference it.

Reply:
https://github.com/kubecost/cost-analyzer-helm-chart/blob/develop/cost-analyzer/values.yaml#L11

This will now enable Grafana for EKS values which I think is fine. To have not done so was an oversight, but as you called out, this absolutely should make the release notes.

Reply:
Does that mean Grafana image need to posted to ECR? I think AWS wants all images to come from there..

Reply:
The requirement from AWS that all images come from ECR is specific to the EKS add-on installed via the Marketplace listing, and in that case it has already been done. I'll just add that for the public ECR instance, we don't push all the images there today, just most.

Reply:
Good to approve @dwbrown2 ?

Reply:
Putting defaults back as suggested by Jesse.